### PR TITLE
Math-Magicians: Full test

### PR DESCRIPTION
**Pull request by Nelitaa**
**Project requirements**

- We add tests to the Math Magicians app using [Jest](https://jestjs.io/) and [React Testing Library](https://testing-library.com/docs/react-testing-library/intro).

- We write unit tests for the files operate.js and calculate.js using Jest.

- We create unit tests for all React components:
1. Using Jest snapshots to test the components.
2. Using React Testing Library to simulate user interaction.

**Team:**

1. @hopeforkoami 
2. @Nelitaa

### CODE REVIEW

File: src/tests/Calcul.test.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { render } from '@testing-library/react';
```
Comment (gemmen29) [inline src/tests/Calcul.test.js]: - [x] Kindly add tests that simulate user interaction as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md#:~:text=Use%20React%20Testing%20Library%20to%20simulate%20user%20interaction.).

File: src/tests/calculate.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import calculate from '../logic/calculate';
+
```
Comment (gemmen29) [inline src/tests/calculate.test.js:2]: - [x] Kindly test all functionality and scenarios for the `calculate` file as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md#:~:text=Write%20unit%20tests%20for%20the%20files%20operate.js%20and%20calculate.js%20using%20Jest.).

File: src/tests/Calcul.test.js (hunk 1 above)
Comment (gemmen29) [inline src/tests/Calcul.test.js]: - [x] Kindly fix the failed test
image
as all tests should be executed successfully.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/Calcul.test.js (hunk 1 above)
Comment (Nelitaa) [reply src/tests/Calcul.test.js]: Thank you very much for the observation! My coding partner and I solved the problem 😁

File: src/tests/calculate.test.js (hunk 2 above)
Comment (Nelitaa) [reply src/tests/calculate.test.js:2]: Thank you very much for the observation! My coding partner and I update the test for the calculate.js file 😁

**Review by BenMukebo — APPROVED** [review]
### Hi @Nelitaa, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 

Screenshot from 2022-12-08 21-29-26



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] ``` ## Authors ``` Update your Readme, This is a group project. Please let's include all the authors in the README file. This makes the README file professional and descriptive. 👨🏻‍🎨 
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**